attributes/default.rb

```diff
@@ -1,15 +1,15 @@
-# rubocop:disable Metrics/LineLength
 class ::Chef::Node::Attribute
   include ::Foreman
 end
 
 default['foreman']['path'] = '/usr/share/foreman'
-default['foreman']['version'] = 'stable'
+default['foreman']['version'] = '2.1.4-1'
 default['foreman']['config_path'] = '/etc/foreman'
 
 default['foreman']['use_repo'] = true
 default['foreman']['repo']['uri'] = 'http://deb.theforeman.org/'
-default['foreman']['repo']['components'] = ['stable']
+component_version = node['foreman']['version'].split('.')[0..1].join('.')
```

Comment:
This will probably not work when the version is overwritten above.

Reply:
Lots will probably not work 😝 Given the already start of a table flip, i'm hoping to kinda get it working on the latest versions of cookbooks first then start converting it to custom resources and not worry about that kinda stuff. Step 1) make it work step 2) make it pretty step 3) make it stable.

Appreciate the early feedback though keep it coming. will give me some good things to adjust as i power through this.